src/config/mod.rs

```diff
@@ -1330,6 +1336,7 @@ impl Default for DbConfig {
             enable_unordered_write: false,
             allow_concurrent_memtable_write: None,
             write_buffer_limit: None,
+            lock_write_buffer_limit: None,
```

Comment:
Putting it in CfConfig makes more sense.

Reply:
Done